### PR TITLE
Translation 06_01_CIEL06_3.txt

### PR DESCRIPTION
**Pull request by Gekimon**

### CODE REVIEW

File: script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+"...Ahh, I have a bit of a fever to be honest."//「……ああ。実は少し熱っぽい」
+I wanted to reply that it wasn't that bad, but I knew that putting on a brave front wouldn't be of any use. It would only worry Akiha even more.//　これぐらい平気だよ、と返答したかったが、強がっても仕方がない。秋葉に余計気を遣わせるだけだろう。
+<Page4>
+"There is no need for you to apologize. It's not your fault if your body is weak.//「そう謝る必要はありません。お体が優れないのは兄さんの<責|せ><任|い>ではないでしょう。
+Kohaku is... preparing breakfast. I will call Arach-sensei then. Hisui, please contact her. She should be near the tower in the mornings."//　琥珀……は朝食の支度をしていますから、阿良句先生を呼びましょうか。翡翠、連絡をお願い。今朝は<楼閣|ろうかく>の方にいる筈よ」
+"Understood."//「かしこまりました」
+<Page5>
+"Arach-sensei... Why do you want to call her over? Besides, is that doctor really here?"//「阿良句先生って……あの人を呼んでどうするんだよ。というか、いるの、あの博士？」
+<Page6>
+"...Nii-san. Please, I beseech you not to address her as 'doctor'. It'll go to her head and she'll  completely go haywire."//「……兄さん。くれぐれも先生をハカセ、などと呼ばないように。調子に乗ってしばらく正気を失いますから」I'm not sure if 'go haywire' is too informal or not. If so, there are plenty of alternatives for that.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt:19]: double space after "she'll"

I think 'go haywire' is perfectly acceptable.

File: script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+<Page7>
+"Arach-sensei spent the night here. Since she hasn't yet left, I'll have her give you a check-up, Nii-san.//「阿良句先生は昨夜うちに泊まっていたんです。まだ帰っていないようだし、兄さんの体を診てもらいましょう。
+In spite of appearance, she really is a doctor specialized in surgery. Now that you are living with us, you will see her frequently, so take this as an opportunity to get to know her better."//　あの方、ああ見えて外科の医師でもあるんです。ここで暮らすのなら頻繁に会う人ですから、これを機に親睦を深めてください」surgery 外科 as opposed to internal medicine 内科
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt:22]: specializing in

File: script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+I can't bring myself to say that I'm not injured.//　別に怪我なんてしてないぞ、とは言えなかった。
+I have bruises all over, hidden under my clothes.//　服の下はそれこそ擦り傷だらけである。
+The problem is that if I have my check-up done here, Akiha will see everything. //　問題は、ここで診察を受けたら秋葉に傷がバレるという事だ。
+<Page9>
+"I'll go see her for the check-up instead. She's at near the tower right?//「診察なら俺の方から行くよ。あの人、ロウカクにいるんだな」
+Still, we have something like that here?"//　でも、そんなのうちにあったっけ？
+<Page10>
+"Here is just fine. The tower is far-away after all.
+She'll come right away and simply take your temperature and pulse.# And if you need something against your fever, she'll... no, if you need medication, it'd be better to have Kohaku take care of that.
+Nii-san? Is something the matter? You seem restless. Could it be that, even at your age, you're afraid of  the doctor?"
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt:34]: double space after 'of'

File: script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+<Page11>
+"No, that's not it, I'm used to doctors."//「診察には慣れている。そういうコトじゃなくて」
+"Haa, then is it Arach-sensei who makes you uneasy?"//「はあ。では、阿良句先生が苦手とか？」
+"I wouldn't say that she makes me uneasy, more like she's the type of person I can't really keep pace with. Anyway, that's not the problem."//「苦手というかついて行けない系だろあの人は。だから、そういう話でもなくて」"She's the kind of person that I find a little overwhelming"
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt:38]: type of person I can't keep up with

File: script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+<Page15>
+"No, there's no way I would entirely strip. Usually, I just remove the top.//「いや、さすがに全裸にはならないだろ。普通は上だけ」
+"R-Right. Just the upper garment then?"//「で、ですよね。上着だけ、というコトでしょうか？」
+She makes a grimace, as if she wanted to say something else.//　それ以外に何があるんだ、と顔をしかめる。
+<Page16>
+After her coughing fit, Akiha stands up from the sofa. It looks like she sensed the mood and chose to leave the room.//　秋葉はちょっと咽せた後、無言でソファーから立ち上がった。どうやら空気を読んで退室してくれるようだ。
+<Page17>
+"Humph...//「……コホン。
+I will take my leave. Please feel free to join me in the dining room when you're done with your check-up.//　席を外しますから、診察が終わり次第食堂にどうぞ。I'm not sure she goes to the dining room actually, change to just go to the dining room if not
+I will be out this afternoon as I will have to be attending a business diner. I trust that you will behave yourself in my absence."//　私は午後から外せない会食があるので外出しますが、私が留守だからといって羽目を外さないようお願いします」
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt:59]: as I will be attending a business dinner.

**Review by Avuxo — COMMENTED** [review]
will finish this file in the morning

File: script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+Akiha leaves the room.//　退室していく秋葉。
+That was close. Putting aside what had been said, I'm glad that the matter ended with Akiha none the wiser.//　良かった。会話の内容はどうあれ、秋葉に体の事を知られずに済みそうだ。
+<Page19>
+I wait by myself in the living room.//　ひとり居間で待つ。
+After around ten minutes, I catch a glimpse of Arach-shi through the window looking out onto the courtyard.//　10分ほど時間を潰していると、中庭に面した窓越しに阿良句氏の姿が見えた。
+She is cutting across the courtyard with a lively gait that was perhaps reflecting her good mood.//　ご機嫌なのか、陽気なステップで中庭を突っ切ってくる。
+...If she's coming from the courtyard, does that mean that the tower is located outside the mansion?//　……あの中庭からやってくるという事は、楼閣とやらは屋敷の外にあるのだろうか……？
+<Page20>
+"Hiiii, good mooooorning!//「ハァーイ、おっはよっうチャーン！
+Wanting to get your body touched in the morning,# Shiki-chan, how lustful―――!# I may become victim to the adolescent boy syndrome, like a prey boiling in a stew!" //　朝からカラダをいじってほしいとかぁ、志貴チャンったら超インラーンーーー！　アタシ、シチューみたいにたぎった思春期少年病の餌食になっちゃうのかしらぁん！」A nice Arach line as we like them
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt:70]: I'm sorry, what?

File: script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+<Page21>
+Without taking the front door, Arach-shi directly enters the living room from the terrace.//　阿良句氏は屋敷に入らず、テラスから居間に直接入って来た。
+<Page22>
+"Sorry. I don't feel particularly unwell, and my migraine has already cleared, so can I just go?"//「すみません。別に悪いところはないし目眩も治まったんで帰ってもらっていいですか？」
+<Page23>
+"Aaah, Shiki-chan, you're so stubborn! Don't play so hard to get nooow!# But maybe that's the mark of a first-class survivor? After all, we live in a world where it's natural for those who don't know better to die, right? No matter how many <tries|b a l l s> I get, that wouldn't be enough to impregnate the snack in front of me, huh?# Kyaaaa, as expected of an omnivore's instinct to avoid pregnancy!"//「ああん、志貴チャンったらカタブツちゃーん！　身持ち堅いとか今どきはやんなァーい！　でも、そんなトコロが一流のサバイバーなのかしら？　情弱死すべしな世の中ですし？　目の前のご馳走をホイホイ孕ませてたら<命|タマ>がいくつあっても足りないってコトかしらー！　キャー、さっすが雑食動物、本能のようにヒニンしてるぅー！」Pain.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt:76]: Beg your pardon?

File: script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+<Page30>
+"Y-Yes. Ah, wait, my underwear?"//「は、はい。あ、いや、パンツになるって、はい？」
+"Yes yes, your underwear. Take off all of your clothes except your underwear. Matriarch-chan's already left the room right? Stripping down to your underpants is fiiine. I like that. Or you can get naked if you'd prefer?"//「パンツはパンツじゃん。ようはパンツだけになれって話。当主チャン、それで退室してるんデショ？　いいじゃんパンイチ。アタシ好きダナー。なんならゼンラでもいいわよぅ？」
+"...Uh, are you sure you haven't been drinking?"//「……あの。お酒、呑んでませんよね？」
+<Page31>
+"Ehh, I don't drink. I'm serious all the tiiiime!# That being said, I'm actually high, really high. I've been secreting beta-endorphin since yesterday night. Like it's enriching my life? Recently it's all been routine work. I really need some ruckus in my life. So you've been a great help on that front, Shiki-chan.# Sooo, today I'll check you up really good!"# "W-Wha... When did I undressed!?"//「えー！　飲んでないわよぅ！　アタシいつも真剣だもーん！　と言いたいところだけど、実はハイなの、すごくハイ。昨日の夜から脳内麻薬分泌しまくり。私生活が充実してきたっていうの？　ここんところルーチンワークばっかりだったからさー。やっぱり生活には波乱が必要よね。そのあたり志貴チャンには大感謝。だからぁ、今朝は特別にきっちり診てあげましょう！」「って、いつの間にか脱げてる!?」
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt:101]: beta-endorphin**s**

When did I **get** undressed

also wtf

File: script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+<Page32>
+I don't know how experienced she was, but Doctor Arach already had me take my upper clothes off.//　どれほど場慣れしているのか、阿良句博士はもう俺の上着を脱がせていた。
+Resistance is futile... No, if I resist, she won't be able to continue.//　抵抗は無意味……いや、抵抗すると診察が進まない。
+I only wanted to avoid getting naked and I somehow managed to pass through Doctor Arach's devil hands.//　俺は裸になるのだけは避ける方向で、なんとか阿良句博士の魔の手をくぐり抜けた。
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 6/06_01_CIEL06_3.txt:105]: I somehow managed to fall into Doctor Arach's evil clutches.

**Review by Avuxo — APPROVED** [review]
full review

**Comment by Gekimon** [comment]
I reuploaded the file to put it to the new format and I took account of Ben's comments

**Comment by rschlaikjer** [comment]
The lint is failing on this one since the rest of the repo (including the code to do the linting) is still old - you'll want to rebase main (and put this file in the proper directory) to make it happy. Or just create a new PR if that's easier.

**Pull request closed without merging**